tests/foreman/cli/test_activationkey.py

```diff
@@ -1248,20 +1248,21 @@ def test_positive_content_override(self):
             u'id': result['activationkey-id'],
             u'organization-id': self.org['id'],
         })
-        for override_value in (u'1', u'0'):
+        for override_value in (True, False):
             with self.subTest(override_value):
                 ActivationKey.content_override({
                     u'content-label': content[0]['label'],
                     u'id': result['activationkey-id'],
                     u'organization-id': self.org['id'],
-                    u'value': override_value,
+                    u'value': int(override_value),
                 })
                 # Retrieve the product content enabled flag
                 content = ActivationKey.product_content({
                     u'id': result['activationkey-id'],
                     u'organization-id': self.org['id'],
                 })
-                self.assertEqual(content[0]['enabled?'], override_value)
+                self.assertEqual(
+                    content[0]['enabled?'], str(override_value).lower())
```

Comment:
Currently returned state is `'true'` or `'false'`, not `'1'` or `'0'` anymore, updating it accordingly.